MeetupPlatformApi/Features/Meetups/UpdateMeetup/UpdateMeetupFeature.cs

```diff
@@ -24,6 +26,7 @@ public UpdateMeetupFeature(ApplicationContext context, IMapper mapper)
     /// <response code="204">If updating was successful.</response>
     /// <response code="404">If needed meetup is null.</response>
     [HttpPut("/api/meetups/{id:guid}")]
+    [Authorize(Roles = Roles.Organizer)]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> UpdateMeetup([FromRoute] Guid id, [FromBody] UpdateMeetupDto updateDto)
```

Comment:
where is [ProducesResponseType(StatusCodes.Status403Forbidden)]

Reply:
There is no need to process this code here, since the swagger settings allows to do this automatically

Reply:
I'm talking about **[ProducesResponseType(StatusCodes.Status403Forbidden)]** for 401 and 403 errors